devstack_romana.md

```diff
@@ -0,0 +1,93 @@
+# Using Romana on Devstack
+
+When installation completes, some SSH commands are provided for logging into your new hosts, eg:
+```bash
+ssh -i /.../romana-install/romana_id_rsa ubuntu@52.xx.yy.zz
+ssh -i /.../romana/romana-install/romana_id_rsa ubuntu@54.zz.yy.xx
+```
+
+These can be used to log into the host using the generated SSH key. Use the SSH command for the controller to log in.
+
+## Look up information
+
+A `romana` tool has been provided that lets you see details about the setup, and make changes.
+```sh-session
+ubuntu@ip-192-168-99-10:~$ romana show-hosts
+Listing 2 host(s)
+ip-192-168-99-10
+ip-192-168-99-11
+ubuntu@ip-192-168-99-10:~$ romana show-host ip-192-168-99-10
+ip-192-168-99-10       192.168.99.10        10.0.0.1/16     0
+ubuntu@ip-192-168-99-10:~$ romana show-host ip-192-168-99-11
+ip-192-168-99-11       192.168.99.11        10.1.0.1/16     0
+```
+By default, we have two hosts (`192.168.99.10` and `192.168.99.11`). Each host has its own CIDR (`10.0.0.1/16`, `10.1.0.1/16`) that it will use when allocating IP addresses.
+
+```sh-session
+ubuntu@ip-192-168-99-10:~$ romana show-tenants
+Listing 2 tenant(s)
+c858ac6ab9534f8f86f2fe9a731277e1 (admin)
+7b593643644243e48915acd98df7b0d0 (demo)
+ubuntu@ip-192-168-99-10:~$ romana show-tenant admin
+Tenant:c858ac6ab9534f8f86f2fe9a731277e1 (admin)
+Segments: default, s1, s2
+```
+
+We've also created `admin` and `demo` tenants, with segments. If the `admin` user creates a new instance, it needs to provide a named segment.
```

Comment:
I think the segment names are the same for both tenants, right? That's a bit unfortunate. I'd say:

"We have also created admin and demo tenants. With Romana, a tenant also creates a number of network segments. Those segments are used by Romana to enable access control and network policies. Each segment can contain multiple OpenStack guests. For your convenience, we have pre-created some segments for our two tenants. The segments for both tenants are called "default", "s1" and "s2". However, each tenant can choose their own segment names. Segments are completely independent from each other."